src/Navbar.jsx

```diff
@@ -1,36 +1,44 @@
 import { Link } from 'react-router-dom';
 import PropTypes from 'prop-types';
+import { IconButton, Button } from '@chakra-ui/react';
+import {ChevronLeftIcon} from '@chakra-ui/icons';
+import { useNavigate } from 'react-router-dom';
 
-function Navbar({ isAdmin }) {
+// Navbar is a container with a single button for Contact Us
+
+function Navbar({ showContactUs, title, showSettings, excelDownload, backButton}) { // what is title
```

Comment:
🚫 **[eslint]** <[react/prop-types](https://github.com/jsx-eslint/eslint-plugin-react/tree/master/docs/rules/prop-types.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'excelDownload' is missing in props validation